docs/bee-developers/useful-dev-info.md

```diff
@@ -7,7 +7,7 @@ Welcome to the Dev area! We love PR's! 🐝
 
 We would would love you to get involved with our [Github repo](https://github.com/ethersphere/bee).
 
-All the action can be found in our Mattermost instance, [The Beehive](https://beehive.ethswarm.org/). Sign up and get involved with our buzzing hive of daily dev chat.
+All the action can be found in our Mattermost instance, [our Discord](https://discord.gg/wdghaQsGq5). Sign up and get involved with our buzzing hive of daily dev chat.
```

Comment:
I would also remove the reference to Mattermost if the link is changed to Discord (Mattermost is the name of the chat software that is running under Beehive)